src/components/LandingPage/LandingPage.js

```diff
@@ -60,6 +61,8 @@ class LandingPage extends Component {
               <VoteButtonModal />
             </div>
             {this.props.reduxState.voteSubmissionModalReducer ? <VoteSubmissionModal></VoteSubmissionModal> : <></>}
```

Comment:
I think we should add comments describing any ternary operators -- we can totally add that later though

Reply:
This is also a note to myself lol